.travis.yml

```diff
@@ -304,6 +310,12 @@ install:
         python@$TRAVIS_PYTHON_VERSION
         $COMPILERSPEC &&
       spack load python@$TRAVIS_PYTHON_VERSION $COMPILERSPEC;
+      if [ "$USE_INTERNAL_PYBIND11" == "OFF" ]; then
+        travis_wait spack install
+          py-pybind11@2.2.1 ^python@$TRAVIS_PYTHON_VERSION
```

Comment:
In the heat of the moment, you probably forgot that you bumped the minimum.
```py-pybind11@2.2.3```

Reply:
copy-pasted from the diff when re-adding it xD